e2e/content-services/social/social.component.e2e.ts

```diff
@@ -0,0 +1,191 @@
+    });
+
+    afterAll(async (done) => {
+        await uploadActions.deleteFilesOrFolder(this.alfrescoJsApi, emptyFile.entry.id);
+        done();
+    });
+
+    describe('Social operations by users on their own components', () => {
+
+        beforeAll(() => {
+            loginPage.loginToContentServicesUsingUserModel(componentOwner);
+        });
+
+        beforeEach(async () => {
+            await navigationBarPage.clickSocialButton();
+        });
+
+        it('[C203006] Should be able to like and unlike their components but not rate them,', () => {
+            socialPage.writeCustomNodeId(emptyFile.entry.id);
+            expect(socialPage.getNodeIdFieldValue()).toEqual(emptyFile.entry.id);
+            likePage.clickLike();
+            expect(likePage.getLikeCounter()).toBe('1');
+            likePage.removeHoverFromLikeButton();
+            expect(likePage.getLikedIconColor()).toBe(blueLikeColor);
+            ratePage.rateComponent(4);
+            expect(ratePage.isNotStarRated(4));
+            expect(ratePage.getUnratedStarColor(4)).toBe(greyLikeColor);
+            likePage.clickUnlike();
+            expect(likePage.getLikeCounter()).toBe('0');
+            likePage.removeHoverFromLikeButton();
+            expect(likePage.getUnLikedIconColor()).toBe(greyLikeColor);
+        });
+
+    });
+
+    describe('Social operations on components that belong to other users', () => {
+        beforeAll(() => {
+            loginPage.loginToContentServicesUsingUserModel(componentVisitor);
+        });
+
+        beforeEach(async () => {
+            await navigationBarPage.clickSocialButton();
+        });
+
+        it('[C260324] Should be able to like, unlike and rate component', () => {
+            socialPage.writeCustomNodeId(emptyFile.entry.id);
+            expect(socialPage.getNodeIdFieldValue()).toEqual(emptyFile.entry.id);
+            likePage.clickLike();
+            expect(likePage.getLikeCounter()).toBe('1');
+            likePage.removeHoverFromLikeButton();
+            expect(likePage.getLikedIconColor()).toBe(blueLikeColor);
+            likePage.clickUnlike();
+            expect(likePage.getLikeCounter()).toBe('0');
+            likePage.removeHoverFromLikeButton();
+            expect(likePage.getUnLikedIconColor()).toBe(greyLikeColor);
+            ratePage.rateComponent(4);
+            expect(ratePage.isStarRated(4));
+            expect(ratePage.getRatedStarColor(4)).toBe(yellowRatedStarColor);
+            ratePage.removeRating(4);
+            expect(ratePage.isNotStarRated(4));
+        });
+
+    });
+
+    describe('Social operations by multiple users on other s components', () => {
+
+        beforeAll(() => {
+            loginPage.loginToContentServicesUsingUserModel(componentVisitor);
+        });
+
+        beforeEach(async () => {
+            await navigationBarPage.clickSocialButton();
+        });
+
+        it('[C260327] Should be able to display total likes and average rating', async () => {
```

Comment:
can we not break this in to smaller tests? if something fails its going to be hard to debug

Reply:
also, might be a good idea to use `expect(something).toBe(something, 'is not something')`